reviews/utils/review_actions.py

```diff
@@ -183,10 +183,9 @@ def is_available(self):
         if not settings.GROUP_SECRETARY in user_groups:
             return False
 
-        if self.review.stage not in [
-            Review.Continuations.REVISION,
-            Review.Continuations.NO_GO,
+        if self.review.stage in [
             Review.Stages.CLOSING,
+            Review.Stages.CLOSED,
```

Comment:
The blame history of this logic is a mess, lol. How did continuations get in here? This should have been caught by someone.

Well, it's been caught now at least. Good find.

In my opinion, this action should be available to the secretary in the following stages:

* Assignment (obviously)
* Commission
* Closing
    * This stage is automatically reached when all opinions have been gathered. But it happens often enough that the secretary wants to add another reviewer before making the final call, so this action should still be available in that stage.

As it stands now, this action will be available in the supervisor stage, which I don't think is right. Could result in weird behaviour if the secretary accidentally adds or removes someone in that stage.